pages/reference.md

```diff
@@ -10,14 +10,15 @@ To achieve this, it is important that the plugin is actually deployed on website
 The plugin can never replace serverside validation and doesn't intend to do so. Having both in place gives you the necessary security for your application, as well as improved usability.
 
 # Markup recommendations
-Each input has a label associated with it: The for-attribute of the label refers to the id-attribute of the input.
+Mandated:  A 'name' attribute is required for all input elements needing validation, and the plugin will not work without this.  A 'name' attribute must also be unique to the form, as this is how the plugin keeps track of all input elements. 
```

Comment:
> Checkbox/radios can have equal names

This is true.  However, a group of radio or checkbox element is treated like a single data input point.  Give two different groups of radio or checkbox elements the same `name` and you''ll have an issue where they're all lumped together incorrectly:

http://jsfiddle.net/0hgyz225/

I'll make some clarifications to note this exception.